### ISSUE TITLE
Spike in loss, dip in mAP at end of warmup

### ISSUE THREAD

**Issue by mpdroid** [ultralytics/yolov5#10209]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I have noticed that there is a spike in training / validation loss + dip in mAP score,   coinciding with end of warmup phase.  I have noticed this in training complete 2 different datasets, with different warmup / learning rate hyper-parameter values. ![results](https://user-images.githubusercontent.com/34389889/202768056-57f989ca-0036-44de-90ae-c41de572399a.png)

Is this normal? I feel like this is impacting the overall end result of the training, and curious if others have experienced this and have found a solution.

**Comment by glenn-jocher** [comment]
@mpdroid it's normal to see loss fluctuate as warmup ends and a normal LR schedule begins. I would train much longer though, i.e. try 50 or 100 epochs.
